api/v1alpha1/edgedevice_types.go

```diff
@@ -44,6 +44,7 @@ type MetricsReceiverConfiguration struct {
 	RequestNumSamples int64  `json:"requestNumSamples,omitempty"`
 	TimeoutSeconds    int64  `json:"timeoutSeconds,omitempty"`
 	URL               string `json:"url,omitempty"`
+	SecretName        string `json:"secretName,omitempty"`
```

Comment:
Please add it to the `EdgeDeviceGroup` as well.

Reply:
@jakub-dzon please provide some context. what is missing? where?

Reply:
Sorry. My bad - I missed that the EdgeDeviceGroup crd file was updated as needed.